Messerli.ChangeCase/Messerli.ChangeCase.csproj

```diff
@@ -6,7 +6,7 @@
         <Product>Messerli.ChangeCase</Product>
         <Description>Transform a string between different casings.</Description>
         <PackageTags>Utility Casing</PackageTags>
-        <Version>1.0.0</Version>
+        <Version>1.1.</Version>
```

Comment:
Something's wrong with this version 👀

Reply:
Fixed in 0a29839b361375e81f121807c6e2515a8ac55254